group_vars/dev_workers.yml

```diff
@@ -22,3 +22,22 @@ docker_users:
   - "{{ galaxy_user.name }}"
 docker_daemon_options:
   data-root: /mnt/docker-data
+
+# cvmfs
+cvmfs_cache_base: /mnt/var/lib/cvmfs
+cvmfs_quota_limit: 20000
+
+# s3fs
+s3fs_cache_base: /mnt/var/lib/s3fs
+s3fs_ensure_diskfree: 80000
```

Comment:
What else would be writing to this disk? If temp files are going here, then the s3fs cache size would be substantially lower than cvmfs?

Reply:
At the moment there is 95083MB free on the disk.  The majority of tmp files will be contained in /mnt/galaxy/tmp/job_working_directory (NFS) instead of on this disk.  Assuming no docker/singularity enabled tools used in testing, nothing else should be writing to this disk.  So maybe it should be `s3fs_ensure_diskfree: 75000`.